.circleci/config.yml

```diff
@@ -227,8 +223,17 @@ jobs:
           at: /tmp/workspace
```

Comment:
Hey, we don't have an executor defined in the job

`executor: aws-ecr/default`

Let's get that in.